android/gradle.properties

```diff
@@ -8,3 +8,4 @@ org.gradle.jvmargs=-Xmx1024m -Xms256m
 android.useAndroidX=true
 android.enableJetifier=true
 android.native.buildOutput=verbose
+supportedLocalizations=ar,be,bg,ca,cs,da,de,el,en,en_GB,es,es_MX,et,eu,fa,fi,fr,fr_CA,he,hi,hu,id,in,it,ja,ko,mr,nb,ne,nl,pl,pt,pt_BR,ro,ru,sk,sv,sw,th,tr,uk,vi,zh_Hans,zh_Hant
```

Comment:
Can we add a comment above? Like
`# Autogenerated by tools/unix/generate_localizations.sh`
?